### ISSUE TITLE
i18n (question)

### ISSUE THREAD

**Issue by VolkerDr** [jumpinjackie/mapguide-react-layout#1482]
I want to change some texts and have seen that some support html tags, others do not (e.g. WMS_NO_LAYER_DESCRIPTION does not). Is it possible that all texts support html tags?

**Comment by jumpinjackie** [comment]
No, it's strictly on a label-by-label basis. All you have to go by is whether the english baseline label has HTML.
